File: 3621-minimum-operations-to-make-array-values-equal-to-k/3621-minimum-operations-to-make-array-values-equal-to-k.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var minOperations = function (nums, k) {
    const st = new Set();
    for (const i of nums) {
        if (i < k) {
            return -1
        } else if (i > k) {
            st.add(i);
        }
    }
    return st.size;
};

File: 3621-minimum-operations-to-make-array-values-equal-to-k/3621-minimum-operations-to-make-array-values-equal-to-k.go
func minOperations(nums []int, k int) int {
	st := make(map[int]bool)
	for _, i := range nums {
		if i < k {
			return -1
		} else if i > k {
			st[i] = true
		}
	}
	return len(st)
}